pom.xml

```diff
@@ -17,14 +17,14 @@
     <powermock.version>2.0.7</powermock.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <raml-module-builder.version>33.0.0</raml-module-builder.version>
-    <vertx.version>4.1.0.CR1</vertx.version>
+    <raml-module-builder.version>33.2.2</raml-module-builder.version>
```

Comment:
We don't need to upgrade to a newer minor version of RMB, just to the latest bugfix version - 33.0.x